dynamic/dynamic.go

```diff
@@ -255,12 +257,19 @@ func (d *Client) connect(ctx context.Context, bus string) (services.DProxy_Conne
 		Type: events.DynamicReplay_AUTH_REQUEST,
 		Payload: &events.DynamicReplay_AuthRequest{
 			AuthRequest: &events.AuthRequest{
-				Token:      d.Token,
-				MessageBus: bus,
+				Token:            d.Token,
+				MessageBus:       bus,
+				PlumberClusterId: d.PlumberClusterID,
+				Name:             d.Options.Name,
 			},
 		},
 	}
 
+	// dProxy needs a unique ID to track the connection
+	// This value can be anything, so long as it is unique
+	md := metadata.Pairs("plumber-stream-id", uuid.NewV4().String())
```

Comment:
Is it okay that it's going to be a diff plumber-stream-id every time? Is that the goal?

Reply:
Yes, it just needs to track it for the lifespan of the `Connect()` gRPC method https://github.com/batchcorp/dproxy/blob/e94c972fefed632ba990eef644f28b43fe394c9d/services/plumberproxy/grpc.go#L85-L88